crates/aptos/src/common/types.rs

```diff
@@ -1918,16 +1918,16 @@ pub struct PoolAddressArgs {
 pub struct MultisigAccount {
     /// The address of the multisig account to interact with
     #[clap(long, value_parser = crate::common::types::load_account_arg)]
-    pub(crate) multisig_address: AccountAddress,
+    pub multisig_address: AccountAddress,
```

Comment:
Do you need to write to this value from `smr_moonshot`? If not, then please keep the current visibility and add a getter method instead. The same goes for the other changes.